src/authentication/components/FlowRow.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import {
+  DataListItemRow,
+  DataListControl,
+  DataListDragButton,
+  DataListItemCells,
+  DataListCell,
+  DataListItem,
+  DataListToggle,
+  Text,
+  TextVariants,
+} from "@patternfly/react-core";
+
+import type AuthenticationExecutionInfoRepresentation from "keycloak-admin/lib/defs/authenticationExecutionInfoRepresentation";
+import type { ExpandableExecution } from "../FlowDetails";
+import { FlowTitle } from "./FlowTitle";
+import { FlowRequirementDropdown } from "./FlowRequirementDropdown";
+
+import "./flow-row.css";
+
+type FlowRowProps = {
+  execution: ExpandableExecution;
+  onRowClick: (execution: ExpandableExecution) => void;
+  onRowChange: (execution: AuthenticationExecutionInfoRepresentation) => void;
+};
+
+export const FlowRow = ({
+  execution,
+  onRowClick,
+  onRowChange,
+}: FlowRowProps) => {
+  const { t } = useTranslation("authentication");
+  const hasSubList =
+    execution.executionList && execution.executionList.length > 0;
```

Comment:
I believe you could fall back on JavaScript casting here.
```suggestion
  const hasSubList = !!execution.executionList.length;
```